content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -1260,10 +1261,7 @@ contributors. Follow the instructions in the SIG-Windows
   to accommodate worker containers crashing or restarting without losing any of
   the networking configuration.
 
-  The "pause" (infrastructure) image is hosted on Microsoft Container Registry
-  (MCR). You can access it using `mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
-  For more details, see the
-  [DOCKERFILE](https://github.com/kubernetes-sigs/windows-testing/blob/master/images/pause/Dockerfile).
+  Refer to the [pause image section](#Pause-Image) to find the latest version of the pause image.
```

Comment:
```suggestion
  Refer to the [pause image](#pause-image) section to find the recommended version
  of the pause image.
```
(fragment identifiers are case sensitive in conformant browsers)

Reply:
Ah, I was following the case in in the rendered markdown https://github.com/kubernetes/website/blob/main/content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md#pause-image but when I copy/paste the link it does show up in all lowercase.

TIL :)